packages/vscode-extension/src/debug/runIconHandler.ts

```diff
@@ -1,14 +1,25 @@
-import { Result, FxError, err, ok, returnUserError } from "@microsoft/teamsfx-api";
+import {
+  Result,
+  FxError,
+  err,
+  ok,
+  returnUserError,
+  ConfigFolderName,
+} from "@microsoft/teamsfx-api";
 import { isValidProject } from "@microsoft/teamsfx-core";
 import { ext } from "../extensionVariables";
 import { ExtensionErrors, ExtensionSource } from "../error";
 import * as vscode from "vscode";
 import * as StringResources from "../resources/Strings.json";
+import * as fs from "fs-extra";
+import * as path from "path";
+import { ExtTelemetry } from "../telemetry/extTelemetry";
+import { TelemetryEvent, TelemetryProperty } from "../telemetry/extTelemetryEvents";
 
-export async function selectAndDebug(args?: any[]): Promise<Result<null, FxError>> {
+export async function selectAndDebug(): Promise<Result<null, FxError>> {
   if (ext.workspaceUri && isValidProject(ext.workspaceUri.fsPath)) {
-    vscode.commands.executeCommand("workbench.view.debug");
-    vscode.commands.executeCommand("workbench.action.debug.selectandstart");
+    await vscode.commands.executeCommand("workbench.view.debug");
```

Comment:
Do we need to await here, what is the point?

Reply:
Here we execute two commands, if we do not use await, the command `workbench.view.debug` may be execute later than `workbench.action.debug.selectandstart`, which may case the options to be closed.